### DISCUSSION TITLE
import template in template

### DISCUSSION THREAD

**Discussion by gxanshu** [pdfme/pdfme#150] [Q&A]
Hi there,

i wanna ask something, let suppose i have a basePDF with 5cm height and 15cm width and i created a template in [designer](https://pdfme.com/template-design) page, added some schemas and saved it in template.json file. after that i created another template with basePDF of 10cm height and 15cm width so now i wanna import that that template.json and in that designer and then fill the schemas of template.json.

how can i do that ?

**Comment by hand-dot** [comment]
Do you wanna merge two templates?

**Reply by gxanshu** [reply]
Yes

**Reply by hand-dot** [reply]
basePDF is one PDF file.
So, First, you have to merge two PDF files into one PDF file, and then import it as basePdf.

**Reply by gxanshu** [reply]
then i can't able to change the position of that PDF file in UI editior

**Reply by hand-dot** [reply]
I see... Sorry.

Could you please send me a screen record?

**Reply by gxanshu** [reply]
i can't able to record because there is no way to add a small size PDF page in a big size PDF. after all with pdfme's designer we only able to edit schemas, we can't edit basepdf.

so i think there is no way to do this.
